linux_os/guide/system/network/network-ipv6/configuring_ipv6/network_ipv6_privacy_extensions/tests/default.fail.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-#
+# platform = Red Hat Enterprise Linux 8
```

Comment:
This will also exclude the rule in other products like OL8. I'm not sure that this is really needed. The following products/profiles do have this rule in them:

```
$ rg network_ipv6_privacy_extensions build/*/profiles -l
build/rhel8/profiles/default.profile
build/fedora/profiles/default.profile
build/rhv4/profiles/rhvh-stig.profile
build/ol8/profiles/default.profile
build/ol7/profiles/ncp.profile
```